### ISSUE TITLE
As a User, I'd like to be able to sign up for an account

### ISSUE THREAD

**Issue by robyurkowski** [grounded/afterburnercms#30]
As an addendum to #26, I should be able to sign up for an account without an admin being forced to create one for me -- so long, of course, as the admin has allowed account creation.

**Comment by knewter** [comment]
Do we really want accounts / user sign in to be built in, instead of in the hosting app?  If ABC is to be an engine (and it should be) then the thing we've learned from the last two years of engines is that Forem does it right and Spree/Refinery does it wrong, in terms of auth.  Anyone that has experience with this in disagreement?  @parndt @radar @GeekOnCoffee?

I'm all for there being an engine, and then a light demo app that shows how to do an implementation, to be followed up by an 'installer' that does the implementation for you / sets up devise if you need / etc. like Forem does.

That demo app should have accounts / registration.  The engine itself should not.  The 'integration test dummy app' would just be an implementation, and would have its own auth defined so we could write integration tests around user/admin distinctions still...

**Comment by robyurkowski** [comment]
No, you're absolutely right, IMO, @knewter. This is something that I hear as a frequent feature request though. I think what this needs to be more about is making sure that there's documentation on how to change a backend login redirect to a front-end redirect, is all, and some good documentation on how you'd do this with Clearance (my own personal choice) or Devise.

I'm going to leave this open just for that reason, but anyone doing this should recognize that this is a pretty light ticket.
